app/src/test/java/protect/card_locker/UtilsTest.java

```diff
@@ -14,53 +15,53 @@
 @Config(sdk = 23)
 public class UtilsTest {
     @Test
-    public void parseBalances() {
-        assertEquals("1", Utils.parseCurrency("1", false).toPlainString());
-
-        assertEquals("1", Utils.parseCurrency("1", true).toPlainString());
-        assertEquals("1.00", Utils.parseCurrency("1.00", true).toPlainString());
-        assertEquals("1.00", Utils.parseCurrency("1,00", true).toPlainString());
```

Comment:
This line seem to fail in the tests

Reply:
The comma probably needs to be replaced with a dot in the parser

Reply:
I actually think the tests are just wrong in the first place. I tried to be overly clever.

But right now parsing anything with decimals goes wrong.

It's honestly scary to touch this code, but #953 is a pretty awful bug :(

Reply:
Specifically, I tried to write it so that using . instead of , or the other way around doesn't really matter. But honestly that was probably not very useful to code and I should've just stuck to whatever the device or selected locale does.

Reply:
Okay, it now seems to work. I think writing tests for core Android features is kinda silly anyway so I'll do more manual testing and then probably delete the tests (because then I'd just be testing Android's .parse)